### DISCUSSION TITLE
Is Sunbird RC manifestation of W3C DID and W3C VC ?

### DISCUSSION THREAD

**Discussion by sujitrs** [Sunbird-RC/community#156] [Ideas & Enhancements]
Is Sunbird RC manifestation of [W3C DID](https://w3c.github.io/did-core/) and [W3C VC](https://www.w3.org/TR/vc-data-model/) ?
Is it complete or where can the status be found as in what is implemented and what is not ?

**Comment by parthlawate** [comment]
@dileepbapat can you help with this ?

**Reply by dileepbapat** [reply]
Yes, Sunbird RC uses / implements W3C for credentialing purpose and it is feature complete. DIDs are used for identifying subject etc, however [resolution](https://w3c.github.io/did-core/#did-resolution) is out of scope (at least for now)
